userscript-styles/WKED-self-study.css

```
@@ -12,72 +12,72 @@
+  }
 
-#wkof_ds #ss_quiz .answer input {
-  background-color: var(--ED-surface-5, #bababa);
-  border-color: var(--ED-surface-4, #535353);
-  --ED-text-color: var(--ED-inverted-text, #151515);
-}
+  #wkof_ds #ss_quiz .answer input {
+    background-color: var(--ED-surface-5, #bababa);
+    border-color: var(--ED-surface-4, #535353);
+    --ED-text-color: var(--ED-text-inv, #151515);
+  }
 
-#ss_quiz[data-itype="radical"] .qwrap,
-#ss_quiz .summary .que[title~="Radical"] {
-  background-color: var(--ED-radical-clr, hsl(1, 39%, 44%));
-}
+  #ss_quiz[data-itype="radical"] .qwrap,
+  #ss_quiz .summary .que[title~="Radical"] {
+    background-color: var(--ED-radical, hsl(1, 39%, 44%));
+  }
 
-#ss_quiz[data-itype="kanji"] .qwrap,
-#ss_quiz .summary .que[title~="Kanji"] {
-  background-color: var(--ED-kanji-clr, hsl(225, 23%, 44%));
-}
+  #ss_quiz[data-itype="kanji"] .qwrap,
+  #ss_quiz .summary .que[title~="Kanji"] {
+    background-color: var(--ED-kanji, hsl(225, 23%, 44%));
+  }
 
-#ss_quiz[data-itype="vocabulary"] .qwrap,
-#ss_quiz .summary .que[title~="Vocabulary"] {
-  background-color: var(--ED-vocab-clr, hsl(166, 36%, 32%));
-}
+  #ss_quiz[data-itype="vocabulary"] .qwrap,
+  #ss_quiz .summary .que[title~="Vocabulary"] {
+    background-color: var(--ED-vocab, hsl(166, 36%, 32%));
+  }
 
-#ss_quiz[data-atype="reading"] .atype {
-  color: var(--ED-text-color, #eeeeee);
-  text-shadow: none;
-  border: none;
-  background-color: var(--ED-reading-clr, #282828);
-}
+  #ss_quiz[data-atype="reading"] .atype {
+    color: var(--ED-text-color, #eeeeee);
+    text-shadow: none;
+    border: none;
+    background-color: var(--ED-reading, #282828);
+  }
 
-#ss_quiz[data-atype="meaning"] .atype {
-  color: var(--ED-inverted-text, #151515);
-  text-shadow: none;
-  border: none;
-  background-color: var(--ED-meaning-clr, hsl(36, 56%, 44%));
-  background-image: none;
-}
+  #ss_quiz[data-atype="meaning"] .atype {
+    color: var(--ED-text-inv, #151515);
+    text-shadow: none;
+    border: none;
+    background-color: var(--ED-meaning, hsl(36, 56%, 44%));
+    background-image: none;
+  }
 
-#wkof_ds #ss_quiz[data-result="correct"] .answer input {
-  color: var(--ED-text-clr, #eeeeee) !important;
-  background-color: var(--ED-correct, #58896f) !important;
-}
+  #wkof_ds #ss_quiz[data-result="correct"] .answer input {
+    color: var(--ED-text, #eeeeee) !important;
+    background-color: var(--ED-correct, #58896f) !important;
+  }
 
-#wkof_ds #ss_quiz[data-result="incorrect"] .answer input {
-  color: var(--ED-text-clr, #eeeeee) !important;
-  background-color: var(--ED-incorrect, #9c4644) !important;
-}
+  #wkof_ds #ss_quiz[data-result="incorrect"] .answer input {
+    color: var(--ED-text, #eeeeee) !important;
+    background-color: var(--ED-incorrect, #9c4644) !important;
+  }
 
-#ss_quiz .settings span.active {
-  color: var(--ED-text-color, #eeeeee);
-}
+  #ss_quiz .settings span.active {
+    color: var(--ED-text-color, #eeeeee);
```

`--ED-text-color` insists on not existing
```suggestion
    color: var(--ED-text, #eeeeee);
```